astropy/cosmology/io/model.py

```diff
@@ -20,6 +20,8 @@
 from astropy.modeling import Parameter as ModelParameter
 from astropy.utils.decorators import classproperty
 
+from .utils import convert_Parameter_to_Model_Parameter
```

Comment:
hmm, should this be all lower-case? After all, you are converting an instance to another instance, not a class to another class.

Reply:
SGTM.